lola.html

```diff
@@ -670,19 +664,92 @@ <h4>Copying “replies” and “inReplyTo” </h4>
       </p>
 
+      <p>
+        If an object lists an Actor/id in "previously", and that actor does not have a movedTo value, or if that movedTo
+        value does not link (possibly in several hops) to the object's current location, then that "previously" value cannot 
+        be trusted.  If a "previously" value cannot be trusted, the expected behavior (though not REQUIRED) is that it is ignored. 
+        Ignoring a "previously" value reduces the situation to an existing common situation, where ActivityPub agents 
+        cannot be sure if an Object had been copied from somewhere else or not.  ActivityPub agents MAY keep a cache of 
+        moved Actor history if they want to be more efficient in resolving and relying on breadcrumbs, although note that
+        a cache that is not carefully designed could prevent a user from recovering from an ill-destined move, by
+        continuing to keep a "movedTo" value that has already been replaced.
+      </p>
+
+      <p><b>Open Issue:</b> More discussion is needed for this proposal to see if it's worthwhile.  The whole point of 
+        breadcrumbs is to "solve" the problem of objects that have been liked moving to a new location, while balancing 
+        tradeoffs such as not overwhelming a network through massive amounts of activity triggered by an Actor's move.  
+        To expand on the problem with a few use cases::
+      </p>
+
+      <ul>
+        <li>Cherry liked Aurora's article in its original location.  When Aurora's article moves, and Cherry sees it in a new 
+          location, it would be nice for Cherry to see that it was already "liked" and not be tempted to "like" it again.</li>
+        <li>Cherry's server hides unwanted content based on a list of blocked Actors, so it would be nice if it could
+          see content that had been moved from blocked Actors and continue to hide this content.</li>
+        <li>That said, it is possible for an ActivityPub server to host false breadcrumbs, so a defensive use case would be that:
+          even when malicious user 'Voldemort' posts an article with a "previously" property including the id of Aurora's article, 
+          Cherry can tell that he did NOT actually "like" this article.
+        </li>
+        <li>Finally, it would be nice if Cherry can reliably tell which articles he already liked even if Aurora's 
+        original server is no longer accessible and even if Voldemort is providing false breadcrumbs.</li>
+      </ul>
+
+      <p>Breadcrumbs on the destination solve the first three use cases in cases where Aurora's original Actor's movedTo 
+        value can be verified. Good caching of moved Actor information could solve the last use case a lot of the time, 
+        but is not guaranteed.
+        In the fourth case, when an ActivityPub agent wants to use information in breadcrumbs but cannot validate them,
+        safe behavior may depend on why it wants to validate the breadcrumb.
```

Comment:
I agree, unverified information is worth storing _somewhere_ to use in, say, private recommendations, blocklists, and content- or follow-suggestions but not for more consequential things like moderation decisions or reputation calculations. 

it's also worth pointing out that these links could be verified with client-side signing of events, or trusted "witness services" (think: internet archive but for fedi), which we can imagine certain huge corporate players and/or Fedi-aaS cos offering down the road as a public good that lifts all boats...